apps/src/applab/applab.js

```diff
@@ -1245,3 +1248,29 @@ var getPegasusHost = function() {
 Applab.isInDesignMode = function () {
   return $('#designWorkspace').is(':visible');
 };
+
+/**
+ * Returns a list of options (optionally filtered by type) for code-mode
+ * asset dropdowns.
+ */
+Applab.getAssetDropdown = function (typeFilter) {
+  var options = assetListStore.list(typeFilter).map(function (asset) {
+    return {
+      text: '"' + clientApi.basePath(asset.filename) + '"',
```

Comment:
I wonder if it would be helpful to create a function quote(str) for these, that returns the string with quotes around it.